contents/handbook/people/team-structure/infrastructure.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Team Infrastructure & Deployments
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+![Image of Cloud Infrastructure](https://github.com/PostHog/posthog-cloud/blob/master/docs/images/infra.png?raw=true)
+
+## People
+
+- [James Greenhill](/handbook/company/team/#james-greenhill-software-engineer) (Team lead, Data/Infra Engineer)
+- [Karl-Aksel Puulmann](/handbook/company/team/#karl-aksel-puulmann-software-engineer) (Full Stack Engineer)
+
+## Mission
+
+Make using and developing for PostHog as reliable as running water. Wherever you want it.
+
+## Responsibilities
+
+- We don't lose events
+- Data is as up to date as possible
+- Engineers always be able to ship and build
+- Fail fast. Fix faster.
+- Ship anywhere
+- Stack performs at speed of click
+- Support Small Teams (and contributors) in building and debugging Posthog
+
+## Customer
+
+- Other Small Teams in making sure they have the tools they need to build
```

Comment:
I'd have 1-2 customers if you can, maybe
- People using cloud
- People self-hosting PostHog

I'm not sure I understand what kind of tools you'd be offering to other small teams? I think we may have a dev experience team at some point but separate from this.

Reply:
Specifically I'm talking about deploying here. Making sure that everyone here can build and deploy whenever they need to.

I'm not a huge fan of such broad customers. Really our _direct_ customers are the developers building PostHog. It's our job to make sure that they have the tools (databases, queues, nodes, infra) they need to build a successful product. It's also our job to deploy what they build to our customers.

It's true that this is all, in the end, to service the downstream user. If we used that definition all teams should be servicing the users of cloud and self-hosted right?